README.md

```diff
@@ -7,38 +7,70 @@ It is very much a work in progress.
 Getting Started
 ---------------
 
-You will need to install and configure [calyx][] and the python bindings for [odgi]. Instructions for installing odgi can be found [here](https://odgi.readthedocs.io/en/latest/rst/installation.html). If you compile odgi from its source, you will need to [edit your python path](https://odgi.readthedocs.io/en/latest/rst/binding/usage.html) to use the python bindings. Since this will only give you the odgi bindings for the most recent version of python, you may need to download the `.so` files from [bioconda](https://odgi.readthedocs.io/en/latest/rst/binding/usage.html) and add them to your `PYTHONPATH` instead.
+First, clone the repo using 
+```git clone https://github.com/cucapra/calyx-pangenome.git```
+Then follow the instructions below to set up `calyx` and `odgi`.
 
-You will need to fetch some useful inputs by typing `make fetch`.
-This downloads some simple [GFA][] data files into the `./test` directory.
+### Installing Dependencies
 
-If you want to build odgi graph files from the GFA files, try something like `make k.og`.
+#### Calyx
+Follow these [instructions](https://docs.calyxir.org/) to install calyx. You will need to configure the calyx driver, `fud`. We recommend using the native calyx interpreter, so once `fud` is set up, run
+```
+fud config stages.interpreter $ABSOLUTE_PATH_TO_CALYX_REPO/target/debug/interp
```

Comment:
To be honest I find Calyx's `config` steps very brittle. I was trying variations of this command to see if I could simplify it, and I now think my setup is messed up. Deleting and re-cloning Calyx has not worked. Regardless, I think it would pay to follow Calyx's style of notation, <full path to Calyx repository>. Now, in the next line, you say that I should actually stop one level _outside_ the Calyx repository. Just wanted to check you on that...

Reply:
Good catch! The next line should be `Users/username/project/calyx/target/debug/interp`. I will also change the notation style.

I also had some issues with configuring calyx, and the issue ended up being with the config file. On havarti, the config file is located at `~/.config/fud/config.toml` for me. You could try deleting this file, which should effectively revert you back to the defaults, and see if it fixes the issues you've been running into?

If you're not using havarti or if this doesn't fix the problem, shoot me a message!